Python/pywarpx/picmi.py

```diff
@@ -444,14 +444,14 @@ def initialize_inputs(self):
         pywarpx.boundary.particle_lo = [self.bc_rmin_particles, self.bc_zmin_particles]
         pywarpx.boundary.particle_hi = [self.bc_rmax_particles, self.bc_zmax_particles]
 
-        if self.moving_window_zvelocity is not None:
-            if np.isscalar(self.moving_window_zvelocity):
-                if self.moving_window_zvelocity !=0:
-                    pywarpx.warpx.do_moving_window = 1
-                    pywarpx.warpx.moving_window_dir = 'z'
-                    pywarpx.warpx.moving_window_v = self.moving_window_zvelocity/constants.c  # in units of the speed of light
-            else:
-                raise Exception('RZ PICMI moving_window_velocity (only available in z direction) should be a scalar')
+        if self.moving_window_velocity is not None and np.any(np.not_equal(self.moving_window_velocity, 0.)):
```

Comment:
In PICMI_CylindricalGrid class, `moving_window_zvelocity` should be changed to `moving_window_velocity`. This has to be done in a new PR in PICMI. Is there anything else needed for this? @dpgrote @EZoni

Reply:
Thank you, Prabhat. I can look at what needs to be done on the PICMI side, in WarpX I think we might still need to update one PICMI test, in particular this line:
https://github.com/ECP-WarpX/WarpX/blob/daa51549ff6de4d20db1b388251a234bac7c2027/Examples/Physics_applications/laser_acceleration/PICMI_inputs_rz.py#L36
Things in CI will work only after the change on the PICMI side is done and this PR is rebased. I will try to address this soon.

Reply:
@prkkumar @dpgrote This should be the PR needed on the PICMI side: https://github.com/picmi-standard/picmi/pull/54.

Reply:
Thanks, Edoardo! I Updated the RZ laser acceleration test. Will rebase the PR once picmi-standard/picmi#54 is merged.